datadog_checks_dev/datadog_checks/dev/tooling/templates/configuration/instances/jmx.yaml

```diff
@@ -10,6 +10,19 @@
   value:
     type: integer
 
+- name: new_gc_metrics
```

Comment:
I think this belongs in `init_config/jmx.yaml` https://github.com/DataDog/integrations-core/blob/87e6f39df8981fc73ea3fe741c60aed467aa2581/datadog_checks_dev/datadog_checks/dev/tooling/templates/configuration/init_config/jmx.yaml#L13-L24